### PR TITLE
Pass search filters to -core.

### PR DESCRIPTION
**Pull request by jmcarp**
[Resolves https://github.com/eregs/notice-and-comment/issues/173]

Depends on https://github.com/eregs/regulations-core/pull/50

### CODE REVIEW

**Comment by cmc333333** [comment]
Heads up, merge conflicts

**Comment by jmcarp** [comment]
Conflicts resolved.

File: regulations/views/partial_search.py [hunk 1]
```diff
@@ -84,17 +63,15 @@ def get_context_data(self, doc_type, **kwargs):
...
+                context['regulation'], page=page, is_root='false',
+                is_subpart='false')
 
         if doc_type == 'cfr':
             transform_context_for_cfr(context, results)
         else:
             transform_context_for_preamble(context, results)
 
         self.add_prev_next(page, context)
-        self.final_context = context
```
Comment (cmc333333) [inline regulations/views/partial_search.py]: Wonder what this guy did.

**Comment by cmc333333** [comment]
Excited to see this!

**Pull request merged**